### ISSUE TITLE
Beacon Init refactoring

### ISSUE THREAD

**Issue by nboufford** [ubc-systopia/pancast-code#35]
Similar to the previous dongle init code, the structure in the beacon is hard to follow. `beacon_start` precedes `beacon_init` etc. Refactoring of the beacon start-up code would be helpful for debugging and readability.

**Comment by aasthakm** [comment]
Addressed in the following commits:
- https://github.com/ubc-systopia/pancast-code/commit/af79b34ef2be2395c39568cb1ae1c6bc3f2961c0
- https://github.com/ubc-systopia/pancast-code/commit/9facf2ab4c158f084781fb615786e5f30ab11675
- https://github.com/ubc-systopia/pancast-code/commit/817e5210aa8be0f0549b8688199704e229936319
- https://github.com/ubc-systopia/pancast-code/commit/95f8b4fb429d0a6f6a1f3c152daac30d125caeb6
- https://github.com/ubc-systopia/pancast-code/commit/0005ccde438851d2ff4792c3a2b231234a2251e1
- https://github.com/ubc-systopia/pancast-code/commit/570d52dbcfb6184287bf45ec02f72c12d6b8cb3d
- https://github.com/ubc-systopia/pancast-code/commit/457e7a570ab05aea96f37d748e3922bec3d79019
- https://github.com/ubc-systopia/pancast-code/commit/8ab790584ae5c6f29a888ccc8659c430f3af9471
- https://github.com/ubc-systopia/pancast-code/commit/3b3211e8019f3a62031c18e0f9095f766806d101
